test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -194,6 +194,30 @@ class ConcurrentHashMapTest extends MapTest {
         assertTrue(s"missing value '$value'", encounteredValues.contains(value))
     }
   }
+
+  @Test def testNewKeySet(): Unit = {
```

Comment:
Instead of this, use the tests in `SetTest`?

Reply:
Added https://github.com/scala-js/scala-js/pull/4367/commits/7c6cd273e35808874878876d8828ae05a0064bf7

I will remove this tests after the `SetTest`-based tests passed.

Reply:
Removed